akka-stream/src/main/scala/akka/stream/io/SslTls.scala

```diff
@@ -380,6 +380,10 @@ sealed trait SslTlsOutbound
  *  - `enabledProtocols` will be passed to `SSLEngine::setEnabledProtocols()`
  *  - `clientAuth` will be passed to `SSLEngine::setWantClientAuth()` or `SSLEngine.setNeedClientAuth()`, respectively
  *  - `sslParameters` will be passed to `SSLEngine::setSSLParameters()`
+ *
+ * Please note that passing `clientAuth = None` means that no change is done
```

Comment:
"Don't change" would only be relevant if there is other code setting that value. Is that the case?

If not, removing this extra bit would simplify the API. However, if something else is changing that setting, it needs to be documented in any case how this setting and other code being able to change the setting relate, i.e. which one would overwrite the other in case this setting is a `Some(ClientAuth)`.